src/app/core/services/label.service.ts

```diff
@@ -152,14 +160,20 @@ export class LabelService {
           remoteLabel.getFormattedName() === label.getFormattedName());
 
       if (nameMatchedLabels.length === 0) {
+
         // Create new Label (Could not find a label with the same name & category)
         this.githubService.createLabel(label.getFormattedName(), label.labelColor);
+
       } else if (nameMatchedLabels.length === 1) {
         if (nameMatchedLabels[0].equals(label)) {
+
           // the label exists exactly as expected -> do nothing
+
         } else {
+
           // the label exists but the color does not match
           this.githubService.updateLabel(label.getFormattedName(), label.labelColor);
+
```

Comment:
These blank lines are being added intentionally?

Reply:
Yes, with the comments, if statements and functions bunched up it was a little harder to read at first glance. Shall I remove the spaces?